conf/application.conf

```diff
@@ -253,6 +253,18 @@ documentation {
     {
       from = "Samples"
       to = "https://developer.lightbend.com/start/?group=play"
+    },
+    {
+      from = "ScalaAnorm"
+      to = "https://playframework.github.io/anorm/"
```

Comment:
Since we are removing Anorm from the official docs: https://github.com/playframework/omnidoc/pull/30